src/style.css

```
@@ -0,0 +1,50 @@
+@font-face {
+	font-family: "Avenir";
+	src: url("./fonts/Avenir.ttc"), url("./fonts/Avenir-Book.ttf"), url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: "Avenir", sans-serif;
+	font-weight: 300;
+}
```

```suggestion
body {
  font-family: "Avenir", sans-serif;
  font-weight: 300;
}
```
ми використовуємо 2 пробіли а не таби
зміни всюди

---

Оке, сделал
Спасибо